src/ui/components/atoms/InfoTooltip/InfoTooltip.tsx

```diff
@@ -0,0 +1,76 @@
+import React, { useState, useRef, useEffect } from 'react';
+import styles from './InfoTooltip.module.scss';
+import TooltipIcon from './TooltipIcon';
+import type { InfoTooltipProps } from './types/IProps';
+
+type TimeoutType = ReturnType<typeof setTimeout>;
+
+function InfoTooltip({
+  content,
+  position = 'top',
+  size = 'medium',
+  className = '',
+  icon,
+  delay = 300,
+  children,
+}: InfoTooltipProps): JSX.Element {
+  const [isVisible, setIsVisible] = useState(false);
+  const [showTimeout, setShowTimeout] = useState<TimeoutType | null>(null);
+  const tooltipRef = useRef<HTMLDivElement>(null);
+  const triggerRef = useRef<HTMLDivElement>(null);
+
+  const showTooltip = () => {
+    if (showTimeout) clearTimeout(showTimeout);
+    const timeout = setTimeout(() => {
+      setIsVisible(true);
+    }, delay);
+    setShowTimeout(timeout);
```

Comment:
There are too much logic here, this component not should be an atom, should a molecule